gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckImplicitDependenciesTask.java

```diff
@@ -56,6 +57,7 @@ public CheckImplicitDependenciesTask() {
         sourceClasses = getProject().getObjects().property(FileCollection.class);
         ignore = getProject().getObjects().setProperty(String.class);
         ignore.set(Collections.emptySet());
+        suggestionConfigurationName = getProject().getObjects().property(String.class);
```

Comment:
Not sure whether it's better to add another property or change the existing `sourceClasses` property to a `SourceSet`. I was worried that might be incorrect since that property is annotated with `@Classpath`.